src/Utils.php

```diff
@@ -1,5 +1,7 @@
 <?hh // strict
 
+include('language/language.php');
```

Comment:
Add this to the autoloads section in the composer.json file. Then you won't have to include it everywhere.